src/main/core/Resources/config/suggested/messenger_prod.yml

```diff
@@ -0,0 +1,26 @@
+framework:
+    messenger:
+        failure_transport: failed
+        transports:
+            async:
+                dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
+
+                # default configuration
+                retry_strategy:
+                    max_retries: 3
+                    # milliseconds delay
+                    delay: 1000
+                    # causes the delay to be higher before each retry
+                    # e.g. 1 second delay, 2 seconds, 4 seconds
+                    multiplier: 2
+                    max_delay: 0
+                    # override all of this with a service that
+                    # implements Symfony\Component\Messenger\Retry\RetryStrategyInterface
+                    # service: null
+
+            routing:
+                'Claroline\MessageBundle\Entity\Message': async
+                'Claroline\CoreBundle\Entity\Task\ScheduledTask': async
+                'Claroline\CoreBundle\Entity\ForumNotification': async
```

Comment:
We will need to find a way to declare this dynamically, otherwise core will have dependency to plugins. To discuss in meeting.

Reply:
Possible solution: https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Messenger/Handler/MessageSubscriberInterface.php
https://symfony.com/doc/current/messenger.html#binding-handlers-to-different-transports